src/main/java/com/continuum/nova/utils/Utils.java

```diff
@@ -15,332 +15,725 @@ static boolean exists(Object objectToCheck) {
 
     public static void initGuiTextureLocations(List<ResourceLocation> locations) {
         addResources(locations,
-                "gui/bars", "gui/book", "gui/demo_background",
-                "gui/icons", "gui/resource_packs", "gui/server_selection",
-                "gui/spectator_widgets", "gui/widgets", "gui/world_selection",
+                "gui/bars",
+				"gui/book",
```

Comment:
Why is this indented so much?

Reply:
Maybe result of automatic replace. Looks normal in IDE

Reply:
Because one line uses spaces, and another uses tabs. Are you doing this automatic? If you do, just set if you want tabs or spaces. In this PR you're doing really inconsistent formatting changes.